package.json

```diff
@@ -10,7 +10,12 @@
     "content": "yarn workspace @weco/content run dev",
     "build-webapps": "echo '@weco/catalogue @weco/content @weco/identity' | xargs -n 1 | xargs -I% yarn workspace % run build",
     "clean": "rimraf '**/.next **/lib'",
-    "clean-deps": "rimraf '**/node_modules'"
+    "clean-deps": "rimraf '**/node_modules'",
+    "catalogue:test": "echo \"⭕️ Running catalogue tests...\" && cd catalogue/webapp && NODE_ENV=test jest --no-cache && echo \"✅️ catalogue tests finished\"",
```

Comment:
How would you feel about having `test:` first? e.g. `test:catalogue`, `test:content`, etc. – feels a bit more natural to me, not sure what others think

Reply:
that does feel more natural, I'll go with that